assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -146,6 +146,8 @@ function DashboardSummaryWidget( { Widget, WidgetReportZero, WidgetReportError }
 						title={ __( 'RPM', 'google-site-kit' ) }
 						datapoint={ period.totals[ 1 ] }
 						datapointUnit={ currencyCode }
+						change={ previousPeriod.totals[ 1 ] - period.totals[ 1 ] }
```

Comment:
Should it not be `current - previous` ? That is [how you had it originally with the utility function](https://github.com/google/site-kit-wp/pull/3214/files/faaebdf7181d8f931af21c0140b52f854a793a46..9f2a3316950ec4da7faba99ca79dc579e71ed822#diff-5cc841e545f161a83da5721c030f752769179ee5ffec24803b63cbae17e947c0L50-L51).